vhdbuilder/publish/sig-version-create.sh

```diff
@@ -16,6 +16,9 @@
 
 MANAGED_IMAGE_NAME="MI_${REGION}_${GALLERY_NAME}_${IMAGEDEFINITION_NAME}_${IMAGE_VERSION}"
 
+# TODO(qinhao): change image version pattern of Windows to ${WINDOWS_VERSION}.$(date +"%y%m%d"), e.g. 17763.1457.200909,
```

Comment:
Ha, nothing except the todo comment, I don't mind close this PR.
Still updated [doc](https://msazure.visualstudio.com/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/84724/windows-VHD-build-and-release) and the image version filed of [release pipeline](https://msazure.visualstudio.com/CloudNativeCompute/_releaseDefinition?definitionId=397&_a=definition-variables)

Reply:
So the version from the VHD building artifacts is the format `17763.1457.200909`, right?

Reply:
An example VHD building artifact vhd-publishing-info.json:
```
    {
        "vhd_url" : "https://winvhdbuildersaceastus.blob.core.windows.net/vhds/aks-1599634754-osDisk.8f4e62d6-7836-491b-b021-e4db2d650542.vhd?sr=c&sp=rl&sv=2018-11-09&st=2020-09-08T00%3A00Z&sig=ApeqbJfGOg7UsPkT3BFLLX%2BJzio/HcaipZ%2BJTCkdq%2B8%3D&se=2021-09-09T00%3A00Z",
        "os_name" : "Windows",
        "sku_name" : "windows-2019",
        "offer_name" : "Windows",
        "windows_version" : "17763.1457"
    }
```

We have windows_version defined in VHD building artifacts, and script publishing image to PIR append `.$(date +"%y%m%d")` to windows_version to be a full version which is visible in PIR.

Reply:
So we can get the full version from release notes and just pass it as IMAGE_VERSION, right?

Reply:
I am not sure it's good to bind the VHD with the image version before a release. But I don't object this idea if it turns out to be a good.

Reply:
It seems like that we cannot get the full version from release notes. So I am ok for the steps in the doc you updated. We may need to pick the full version from some result if we want to automate the vhd publish process.

Reply:
We use full versions to name the release notes.